eksperyment_mikroserwisy.rst

```diff
@@ -0,0 +1,78 @@
+﻿Testowanie mikrousług/Dockera
```

Comment:
Chodzi o to, że na GH wrzuca się komentarz do konkretnej linijki - jeżeli ich nie łamiesz, to wrzucasz komentarz do całego akapitu a nie do fragmentu. Co przy długich akapitach sprawia problem z ustaleniem, do czego komentarz się odnosi.